vulcanai/utils.py

```diff
@@ -1,421 +1,24 @@
-"""Contains auxilliary methods."""
```

Comment:
I think these need to be included before merge, no?

Reply:
Yes they need to be included. This pr was to facilitate reviewing for the changes in the rest of the codes. Sorry about that.